packages/lit-dev-content/site/docs/components/properties.md

```diff
@@ -30,11 +42,26 @@ The component shouldn't change its own public properties, except in response to
 
 Lit also supports _internal reactive state_. Internal reactive state refers to reactive properties that _aren't_ part of the component's API. These properties don't have a corresponding attribute, and are typically marked protected or private in TypeScript.
 
+{% switchable-sample %}
+
 ```ts
 @state()
 private _counter = 0;
 ```
 
+```js
+static properties = {
```

Comment:
Are `@state` decorators handled with `static properties` as well?

Reply:
Yep, but great question because I forgot the `{state: true}` setting here! Thanks, fixed!